src/client/trackers/pageAttention.js

```diff
@@ -0,0 +1,34 @@
+import { broadcast } from '../broadcast';
+import ScrollDepth from './page-attention/ScrollDepth';
+import AttentionTime from './page-attention/AttentionTime';
+
+// TODO: The tracking event data for the `page:interaction` and `page:scrolldepth`
+// events is needlessly different. We should work with the data team to align it.
+export const pageAttention = (options = {}) => {
```

Comment:
should this be added to https://github.com/Financial-Times/n-tracking/blob/master/src/client/trackers/index.js? or is it intended to be imported separately?

Reply:
Good spot, it needs to be made public